android-plugin/RULES.md

```diff
@@ -0,0 +1,38 @@
+# Android-specific Rules
+
+Android-specific rules rely on a multi-scope scanning, including Java source files, Xml files, Gradle files, and more generally the project structure.
+
+### 🍃 Environment
+
+The set of rules comes from the detailed [Energy Smells catalog](https://olegoaer.perso.univ-pau.fr/android-energy-smells/). 3/4 of them have been already implemented in the plugin. Table of unimplemented rules below:
+
+| # | **Rule Name**      |     **Scanner**     |      **Observation**     |
+|---|:----------------|:-------------:|:-------------:|
+| ELEA001 | Everlasting Service        | Java | Requires `PostProjectAnalysisTask()` callback |
```

Comment:
I think that we should explain somewhere what "Requires `PostProjectAnalysisTask()` callback" means.

Reply:
It seems this method appeared in Sonar API 8.9. If so, that's good news, isn't it @Jberque?